README.md

```diff
@@ -1,4 +1,6 @@
 mne-testing-data
 ================
 
-Data for mne-python testing
+Data for mne-python testing.
+
+Use the function `mne.datasets.testing.data_path()` to acquire this data in MNE.
```

Comment:
Can also add "To update to the latest version, use `mne.datasets.testing.data_path(force_update=True, verbose=True)`"

I'd also put `verbose=True` in the line above in case people have set the logging level to WARNING